src/ngraph/type/element_type.cpp

```diff
@@ -80,6 +81,7 @@ static const map<element::Type_t, const TypeInfo>& get_type_info_map()
         {element::Type_t::i16, TypeInfo(16, false, true, false, "int16_t", "i16")},
         {element::Type_t::i32, TypeInfo(32, false, true, true, "int32_t", "i32")},
         {element::Type_t::i64, TypeInfo(64, false, true, false, "int64_t", "i64")},
+        {element::Type_t::u1, TypeInfo(1, false, false, false, "u1", "u1")},
```

Comment:
parameter 5 "u1" is the C type used to represent this number and u1 is not a valid C type.

Reply:
Yes `u1` is not a valid C type, this is normal. What's your proposal? This time we are going to add limited support for u1 only. There is no need to manipulate with u1 values except just storing it in a constant node and consuming in BinaryConvolution. It shouldn't affect any parts of nGraph.